backend/curriculum_tracking/tests/test_Topic_Progress.py

```diff
@@ -0,0 +1,58 @@
+from django.test import TestCase
+from django.utils import timezone
+from datetime import datetime, timedelta
+from unittest.mock import patch
+
+from curriculum_tracking.tests import factories
+from curriculum_tracking.models import AgileCard, ContentItem
+from activity_log.models import LogEntry
+import curriculum_tracking.activity_log_entry_creators as log_creators
+
+
+class TestTopicProgress(TestCase):
+
+    def test_get_card_duration_in_current_column_returns_none_when_there_are_no_log_entries(
```

Comment:
please remove. There is a totally different issue about calculating the duration. This is in the wrong place now.